tests/test_attention_masking.py

```diff
@@ -43,3 +43,44 @@ def test_attention_masking1():
 
     assert torch.allclose(output_no_mask, output_dummy_mask)
     assert torch.allclose(output_no_mask, output_mask_initial[:, 4:])
+
+
+@pytest.mark.gpu
```

Comment:
Is it that slow without gpu? This seems like a test that will be good to run automatically.